HTML/CSS changes to support datepicker addition

skoolbus39 opened the pull request.

skoolbus39 committed:
Remove custom legend styles

skoolbus39 committed:
Initial Create Event form markup updates

skoolbus39 committed:
Remove inline font-size styling

skoolbus39 committed:
Remove font-size class from image upload legend

skoolbus39 committed:
Update Create Event markup

skoolbus39 committed:
Remove styles handled by utility classes

skoolbus39 committed:
Update Create Calendar markup

jsturek commented:
In `/www/templates/default/html/js/manager.js` add  
```
       if ($('#recurring').is(":checked")) {
            $(".recurring-container").show();
        }

        $('#recurring').click(function (click) {
            if ($(this).is(":checked")) {
                $(".recurring-container").show();
            } else {
                $(".recurring-container").hide();
            }
        });
```
to show container when checkbox is check and to handle show/hide on clicks

In `/www/templates/default/html/less/manager.less` update `#recurring ~ .recurring-container {` to just `.recurring-container  {`  and remove the entire block for `#recurring:checked ~ .recurring-container {`  to always hide the container and let the above javascript handle show/hide based on checkbox state.

Will need to run `make` in commad line to compile js and less.

Will need to update `/www/templates/default/html/manager/AddDatetime.tpl.php` with similar form markup updates.

skoolbus39 committed:
Update Less/CSS/JS toggle recurring events details

skoolbus39 committed:
Update AddDatetime.tpl.php markup

skoolbus39 committed:
Reorder attributes, remove commented-out block

Formatted according to Code Guide: https://codeguide.co/#html-attribute-order

skoolbus39 committed:
Update Image Upload markup

- Reorder attributes
- Update indents
- Add checkbox markup and form groups
- Convert required <span>s to <small> elements

skoolbus39 committed:
Update markup for Add User

skoolbus39 committed:
Update Edit User Info markup

skoolbus39 committed:
Update Users markup

skoolbus39 committed:
Update Subscription markup

skoolbus39 committed:
Update Welcome markup

skoolbus39 committed:
Update Edit Account Info markup

skoolbus39 committed:
Update Delete Calendar markup

skoolbus39 committed:
Update Clean Up Calendar markup

skoolbus39 committed:
Update Create Subscription markup

skoolbus39 committed:
Update Recommend markup

skoolbus39 committed:
Update Edit Event markup

skoolbus39 committed:
Update View Event markup

skoolbus39 committed:
Update Recommend checkboxes

skoolbus39 committed:
Update Search markup

- Remove unneeded `dcf-label` class
- Reorder atrributes

skoolbus39 committed:
Fix radio buttons and spacing in Recommend

skoolbus39 committed:
Add missing `for` attribute to label

skoolbus39 committed:
Add missing closing tag bracket

jsturek commented on www/templates/default/html/manager/AddDatetime.tpl.php:
```diff
@@ -50,207 +50,234 @@
-
-            <input class="dcf-input-control" type="checkbox" id="location-save" name="location_save">
-            <label class="dcf-label" for="location-save">Save this location for future events</label>
+        <legend></legend>
```
Probably should remove empty `<legend>`

skoolbus39 replied:
Oops! I meant to add some text for that. Because it's a `<fieldset>` it also needs a `<legend>`. I'll add text.

skoolbus39 committed:
Convert location/date/time info to a table

skoolbus39 committed:
Remove background-color from .edt-record

skoolbus39 committed:
Add missing table heading

skoolbus39 committed:
Update Users markup (button styles)

jsturek commented:
Reminder: `../manager/ViewEvent.tpl.php` still has table margin bottom issue with Contact Info `<h2>`

skoolbus39 committed:
Remove Oxford commas, use ampersands

skoolbus39 committed:
Remove Oxford commas, use ampersands (#2)
